script.js

```diff
@@ -1 +1,65 @@
-// 다들 화이팅!! ٩( *˙0˙*)۶
\ No newline at end of file
+document.addEventListener('DOMContentLoaded', function() {
+    const todoInput = document.getElementById('todo-input'); // 할 일 입력 필드
+    const addTodoBtn = document.getElementById('add-todo-btn'); // 할 일 추가 버튼
+    const todoListBox = document.querySelector('.todo-list-box ul'); // To Do 목록
+    const doneListBox = document.querySelector('.done-list-box ul'); // Done 목록
+    const todoListTitle = document.getElementById('todo-list-title'); // To Do 제목
+    const doneListTitle = document.getElementById('done-list-title'); // Done 제목
+  
+    function updateCount() {
+        // 각 목록 별 갯수 표시 기능
+        todoListTitle.textContent = `📋 To Do (${todoListBox.children.length})`;
+        doneListTitle.textContent = `💿 Done (${doneListBox.children.length})`;
+    }
+  
+    function createTodoItem(text) {
+        // 입력받은 텍스트를 항목으로 만들어주는 함수
+        const li = document.createElement('li');
+        li.textContent = text;
+  
+        const deleteBtn = document.createElement('span');
+        deleteBtn.textContent = ' 🗑️';
+        
+        deleteBtn.addEventListener('click', function(event) {
+            // 항목 삭제 기능
+            event.stopPropagation();
+            li.remove();
+            updateCount(); // 항목 삭제 후 갯수 업데이트
+        });
```

Comment:
화살표 함수도 사용해 보시면 좋을 것 같아요 🙂
```suggestion
        deleteBtn.addEventListener('click', (event) => {
            // 항목 삭제 기능
            event.stopPropagation();
            li.remove();
            updateCount(); // 항목 삭제 후 갯수 업데이트
        });
```

Reply:
화살표 함수가 아직 익숙치 않아서 해당 부분을 놓쳤던 것 같습니다😭
익숙해지도록 연습하고 익혀보겠습니다!